content/ADR-145-pick-realm-v2.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: adr
+adr: 145
+title: Pick Realm Algorithm V2
+date: 2022-11-28
+status: Draft
+type: RFC
+spdx-license: CC0-1.0
+slug: /adr/TEMPLATE
+authors:
+  - agusaldasoro
+---
+
+## Abstract
+
+The logic on the realm scoring algorithm should be improved to take into consideration more variables in order for a server to be more eligible. As with the current model, any added variable needs to be modifiable through Feature Flags so that they can be changed in case of need without a release or a reboot of the server.
```

Comment:
The abstract does not describe the document's current status nor summarize its content.